### DISCUSSION TITLE
Can someone help me understand the crosh daybreak?

### DISCUSSION THREAD

**Discussion by s0urce-c0de** [3kh0/ext-remover#1119] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
For me when I try the terminal method (`ssh a@>crosh`), right click terminal app, click on my ssh, it acts like regular ssh. If this works, why can't we just use the `--dev` flag for crosh to force devmode commands? (Yes it exists, I read the source. I am just too lazy to go find the link. If I have time, I'll go find the link.)

**EDIT:**
I mean like `ssh a@>crosh --dev`

**Comment by S-PScripts** [comment]
closed (one year expiry)
